frontend/index.html

```diff
@@ -8,6 +8,36 @@
   </head>
 
   <body>
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-app.js"></script>
+    
+    <!-- TODO: Add SDKs for Firebase products that you want to use
+         https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-analytics.js"></script>
+    
+    <script>
+      // Concerned Citizen,
+      // We know what you're asking yourself: "Did these brain dead Covid Watch devs just check their API keys in to
+      // an open source repository?". The answer is of course, yes we did. But don't worry, these are not "API keys"
+      // in the normal sense of the word. They should rather be thought of as configuration URLs that tell our app
+      // which Firebase infrastructure it should talk to. Firebase handles security in other ways, see the following
+      // Stack Overflow link for more info: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public
+
+      var firebaseConfig = {
```

Comment:
I'm confused as to why this is being hardcoded (and duplicated) here

Reply:
Yeah that's a good point.  I added the only additional item that wasn't already in `firebase.config.prod.js`, `measurementId`, to `firebase.config.prod.js`.

And I'm not sure if the `<script>` parts of `index.html` are necessary to have there or at all tbh.

And `firebaseApp.analytics()` added to `firebase-repl.js` I think initializes it in the way we want?

Honestly I'm not totally sure where all this should go so if this seems to be the wrong approach plz advise @ibeckermayer .